website/docs/guides/dependencies.md

```diff
@@ -36,6 +36,45 @@ For example:
 org.postgresql:postgresql:42.2.8
 ```
 
+### Excluding Transitive Dependencies
+
+To exclude a transitive dependency of dependency in a Scala CLI project, use the `exclude` parameter when specifying
+the dependency:
+
+- `exclude=org%%name` - for Scala modules
+- `exclude=org%name` - for Java modules
+
+It requires passing the organization and module name of dependency to exclude. For example, let's say you have the following Scala code:
+
+```scala title=Main.scala
+//> using dep "com.lihaoyi::pprint:0.8.1"
+object Main extends App {
+  println("Hello")
+}
+```
+
+For example, if you want to compile it with the `pprint` library but exclude its `sourcecode` dependency. You can use
+the exclude parameter is as follows:
```

Comment:
you used the `For example (...)`phrase in the preceding sentence, so it feels off to find it here again.
```suggestion
If you want to compile it with the `pprint` library but exclude its `sourcecode` dependency, you can use
the `exclude` parameter as follows:
```